model/qti/Element.php

```diff
@@ -737,4 +738,21 @@ private function isDebug(): bool
         }
         return false;
     }
+
+    private function getBodyAttributes(): string
+    {
+        $body = $this->getBody();
+
+        if(!$body instanceof Element) {
+            return '';
+        }
+
+        $attributeValues = $body->getAttributeValues();
+        $bodyAttributes = [];
+        foreach ($attributeValues as $attributeName => $attributeValue) {
+            $bodyAttributes[] = sprintf('%s="%s"', $attributeName, $attributeValue);
```

Comment:
**thought:** Is it safe enough to do so? I don't any sanitisation of the content.